finch.yaml

```diff
@@ -148,27 +148,39 @@ provision:
 - mode: user
   script: |
     #!/bin/bash
-    sudo chown $USER /mnt/lima-finch
 
-    # https://github.com/containerd/nerdctl/blob/cffdf87ff4d648a5344eea1406bb95ca3ad7eaa4/extras/rootless/containerd-rootless.sh#L144-L146
-    # XDG_DATA_HOME & ~/.local/share: https://github.com/containerd/nerdctl/blob/cffdf87ff4d648a5344eea1406bb95ca3ad7eaa4/extras/rootless/containerd-rootless.sh#L51
```

Comment:
Could we add back the references for rootless directories?

Reply:
Updated.